gorillamux/adapter.go

```diff
@@ -0,0 +1,49 @@
+package gorillamux
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/awslabs/aws-lambda-go-api-proxy/core"
+	"github.com/gorilla/mux"
+)
+
+type GorillaMuxAdapter struct {
+	core.RequestAccessor
+	router *mux.Router
+}
+
+func New(router *mux.Router) *GorillaMuxAdapter {
+	return &GorillaMuxAdapter{
+		router: router,
+	}
+}
+
+func newLoggedError(format string, a ...interface{}) error {
```

Comment:
Is this a standard function we should put in the core package and re-use everywhere?